eris.go

```diff
@@ -143,6 +159,14 @@ func (e *rootError) Format(s fmt.State, verb rune) {
 	printError(e, s, verb)
 }
 
+//StackFrames returns a slice of all the frames in the error chain
```

Comment:
Remove this comment here and below (they don't show up in godoc and the comment above the exported StackFrames is already sufficient)